index.d.ts

```diff
@@ -511,6 +511,13 @@ declare namespace algoliasearchHelper {
     ruleContexts?: string[];
     optionalFilters?: Array<string | string[]>;
     queryLanguages?: string[];
+
+    /**
+     * The relevancy threshold to apply to search in a virtual index [0-100]. A Bigger
+     * value means fewer, but more relevant results, smaller value means more, but
+     * less relevant results.
+     */
+    relevancyStrictness?: number;
```

Comment:
this one likely isn't needed

Reply:
Can you tell me why it's not needed?
When we need to set `relevancyStrictness` from the frontend side, we will need it, no?

Reply:
because the interface inherits from the client type, but in this case we still want to copy it for v3 compact

Reply:
Ah got what you mean 👍🏼